solutions/swb-reference/src/environment/sagemakerNotebook/sagemakerNotebookEnvironmentLifecycleService.test.ts

```diff
@@ -129,9 +128,13 @@ describe('SagemakerNotebookEnvironmentLifecycleService', () => {
     expect(response).toEqual({ ...environment, status: 'PENDING' });
   });
 
-  test('Launch should return mocked id when mounting datasets', async () => {
+  test.skip('Launch should return mocked id when mounting datasets', async () => {
     const envHelper = new EnvironmentLifecycleHelper();
-    environment.datasetIds = ['exampleDS'];
+    environment.DATASETS = [
+      {
+        id: 'dataset-123'
```

Comment:
only included the `id` here and not the whole object since the rest isn't needed for this test.